corehq/apps/campaign/tests/test_models.py

```diff
@@ -1,16 +1,22 @@
-import pytest
+from unmagic import fixture, use  # https://github.com/dimagi/pytest-unmagic
 
 from ..models import Dashboard, DashboardMap, DashboardReport, DashboardTab
 
 
-@pytest.fixture
-def dashboard():
-    return Dashboard.objects.create(domain='test-domain')
+@use('db')
+@fixture
+def dashboard_fixture():
+    dashboard = Dashboard.objects.create(domain='test-domain')
+    try:
+        yield dashboard
+    finally:
+        dashboard.delete()
 
 
-@pytest.fixture
-def dashboard_maps(dashboard):
-    return [
+@fixture
+def dashboard_maps():
+    dashboard = dashboard_fixture()
+    yield [
         DashboardMap.objects.create(
```

Comment:
Looks like the value of this fixture is never used directly (the fixture is not called inside a test function that I saw), and therefore this is slightly misleading, although it works just fine. Since the yielded value is not important, what do you think of this instead?
```py
    dashboard = dashboard_fixture()
    DashboardMap.objects.create(
        dashboard=dashboard,
        ...
    )
    DashboardMap.objects.create(
    ...
    yield
```